tests/network/connectivity/conftest.py

```diff
@@ -359,11 +363,38 @@ def vm_ovs_bridge_attached_vmb_destination(
     ]
 
     yield from create_running_vm(
-        name=f"vmb-{OVS_BRIDGE}",
-        end_ip_octet=2,
+        namespace=namespace,
+        client=unprivileged_client,
         node_selector=get_node_selector_dict(node_selector=worker_node2.hostname),
         network_names=network_names,
         ipv6_primary_interface_cloud_init_data=ipv6_primary_interface_cloud_init_data,
-        client=unprivileged_client,
-        namespace=namespace,
+        secondary_interfaces_cloud_init_data=secondary_interfaces_cloud_init_data,
+        name=f"vmb-{OVS_BRIDGE}",
+        host_id=2,
     )
+
+
+@pytest.fixture(scope="class")
+def secondary_interfaces_cloud_init_data(
+    ipv4_supported_cluster: bool, ipv6_supported_cluster: bool
+) -> Callable[[int], dict[str, dict[str, dict[str, list[str]]]]]:
```

Comment:
this should be a simple helper function that gets ipv4_supported etc and returns a dict.